src/services/EthereumServiceTesting.ts

```diff
@@ -0,0 +1,156 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+const ADDRESS = "0xE834627cDE2dC8F55Fe4a26741D3e91527A8a498";
+
+/* eslint-disable no-console */
+import { ethers, Signer } from "ethers";
+import { BaseProvider, Web3Provider } from "@ethersproject/providers";
+import { EventAggregator } from "aurelia-event-aggregator";
+import { autoinject } from "aurelia-framework";
+import { DisclaimerService } from "services/DisclaimerService";
+import { Address, AllowedNetworks, EthereumService, Hash, IBlockInfo, Networks } from "./EthereumService";
+
+@autoinject
+export class EthereumServiceTesting {
```

Comment:
@dkent600 
Main changes can be seen here
3f5eebed0d784dc55d55d311dc229450cd460fde

Rest was to reduce unnecessary content of class